### PR TITLE
[ISSUE #806] code optimization and delete invalid code in eventmesh-e…

### PR DESCRIPTION
**Pull request by fengyongshe**
Fixes ISSUE #806.

### Motivation

code optimization and delete invalid code in eventmesh-examples module

### Modifications

*Use ExampleConstants to hold constants in examples
*Delete invalid code in AsyncSubscribe

### Documentation

- Does this pull request introduce a new feature?  no

### CODE REVIEW

File: eventmesh-examples/src/main/java/org/apache/eventmesh/grpc/sub/app/service/SubService.java [hunk 1]
```diff
@@ -115,8 +115,8 @@ public void cleanup() {
      * Count the message already consumed
      */
     public void consumeMessage(String msg) {
-        logger.info("consume message {}", msg);
+        logger.info("consume message: {}", msg);
         countDownLatch.countDown();
-        logger.info("remaining number of messages to be consumed {}", countDownLatch.getCount());
+        logger.info("remaining number of messages to be consumed : {}", countDownLatch.getCount());
```
Comment (ruanwenjun) [inline eventmesh-examples/src/main/java/org/apache/eventmesh/grpc/sub/app/service/SubService.java:120]: For  code style consistency, you can delete this space.
```suggestion
        logger.info("remaining number of messages to be consumed: {}", countDownLatch.getCount());
```

File: eventmesh-examples/src/main/java/org/apache/eventmesh/http/demo/pub/eventmeshmessage/SyncRequestInstance.java [hunk 2]
```diff
@@ -71,7 +73,7 @@ public static void main(String[] args) throws Exception {
 
             EventMeshMessage rsp = eventMeshHttpProducer.request(eventMeshMessage, 10000);
             if (logger.isDebugEnabled()) {
-                logger.debug("sendmsg : {}, return : {}, cost:{}ms", eventMeshMessage.getContent(), rsp.getContent(),
+                logger.debug("sendmsg : {}, return : {}, cost:{} ms", eventMeshMessage.getContent(), rsp.getContent(),
```
Comment (ruanwenjun) [inline eventmesh-examples/src/main/java/org/apache/eventmesh/http/demo/pub/eventmeshmessage/SyncRequestInstance.java:76]:
```suggestion
                logger.debug("sendmsg: {}, return: {}, cost:{} ms", eventMeshMessage.getContent(), rsp.getContent(),
```

**Review by ruanwenjun — CHANGES_REQUESTED** [review]
Just some small need to change.

File: eventmesh-examples/src/main/java/org/apache/eventmesh/http/demo/pub/eventmeshmessage/SyncRequestInstance.java (hunk 2 above)
Comment (fengyongshe) [reply eventmesh-examples/src/main/java/org/apache/eventmesh/http/demo/pub/eventmeshmessage/SyncRequestInstance.java:76]: ok

**Review by ruanwenjun — APPROVED** [review]
LGTM

**Pull request merged**